moderation.php

```diff
@@ -3063,20 +3063,22 @@
 				}
 
 				// Add the IP's to the banfilters
-				foreach(array($user['regip'], $user['lastip']) as $ip)
-				{
-					$ip = my_inet_ntop($db->unescape_binary($ip));
-					$query = $db->simple_select("banfilters", "type", "type = 1 AND filter = '".$db->escape_string($ip)."'");
-					if($db->num_rows($query) == 0)
-					{
-						$insert = array(
-							"filter" => $db->escape_string($ip),
-							"type" => 1,
-							"dateline" => TIME_NOW
-						);
-						$db->insert_query("banfilters", $insert);
-					}
-				}
+                if($mybb->settings['purgespammerbanip'] == 1) {
```

Comment:
Nit-pick: Indentation should be tabs rather than spaces.

Reply:
Also the `{` brace should be on a separate line; the PR includes some unrelated whitespace changes that can be dropped.

Reply:
I'm happy enough with the unrelated whitespace changes personally - they're mostly improvements to drop un-needed whitespace which won't cause any harm.

Reply:
It can clutter up git history and code comparison - IMO better to introduce code style Checks and fix all files in one go (presumably 1.9).

Reply:
I think it's resolved, can you check?